app/Models/Project.php

```diff
@@ -669,67 +669,67 @@ public function communities(): BelongsToMany
     }
 
     /**
-     * The consultants that are interested in the project.
+     * The participants that are interested in the project.
      *
      * @return \Illuminate\Database\Eloquent\Relations\BelongsToMany
      */
-    public function interestedConsultants(): BelongsToMany
+    public function interestedCommunityMembers(): BelongsToMany
```

Comment:
I'm not sure if the comment or the function name should be updated, but it's probably better if the match. (i.e. that is participant vs community member). Is the concept that "participant" is used to refer to "community members" in the context of a project, or that "community members" become "participants" once they are added to a project? Or maybe something else.

Reply:
> "community members" become "participants" once they are added to a project

Exactly this.